spec/support/queue_helpers.rb

```diff
@@ -1,10 +1,33 @@
 module QueueHelpers
-  def create_stubbed_queue
+  def create_stubs
```

Comment:
The current implementation also has this problem, but from reading the tests, you wouldn't know that `create_stubs` actually stubs out `Bunny` (via `create_connection`). 

Can we think of a better name for these methods so that it is clear we are doing multiple things?